engine/script/src/script/sys_ddf.proto

```diff
@@ -233,3 +233,18 @@ message SetUpdateFrequency
 {
     required int32 frequency = 1;
 }
+
+/*# resume rendering
+ * Resume rendering.
+ * This message can only be sent to the designated `@system` socket.
+ *
+ * @message
+ * @name resume_rendering
+ * @examples
+ * <pre>
+ * msg.post("@system:", "resume_rendering")
+ * </pre>
+ */
```

Comment:
Who is supposed to send this and when?
If it's recommended to be sent in the listener, I think it should be mentioned there as well in the example.